containers/unit_tests/TestUnorderedMap.hpp

```diff
@@ -480,6 +480,48 @@ TEST(TEST_CATEGORY, UnorderedMap_consistent_size) {
   ASSERT_EQ(2u, m.size());
 }
 
+namespace Impl {
+
+using map_type = Kokkos::UnorderedMap<int, void, Kokkos::DefaultExecutionSpace>;
```

Comment:
Use `TEST_EXECSPACE`?

Reply:
Changed in [dd1bd50](https://github.com/kokkos/kokkos/pull/6073/commits/dd1bd50f4d5b1d191d28dbed23d36895fcafe05f).